src/org/zaproxy/zap/extension/tokengen/TokenGenMessagesTableModel.java

```diff
@@ -117,6 +117,15 @@ public Object getValueAt(int rowIndex, int columnIndex) {
         }
     }
 
+    @Override
+    public boolean isCellEditable(int rowIndex, int columnIndex) {
```

Comment:
setCellSelectionEnabled does allow you to copy individual cells, however it does break the highlight as it goes from a row to that particular cell. 
In the end I've decided to remove this function completely as copying the whole row does contain the value. You only have to 'process' it a bit.